src/Manager/FlatpickrManager.php

```diff
@@ -52,7 +52,12 @@ public function getOptionsFromDca(string $table, string $field): array
         $options = [];
         $dca = &$GLOBALS['TL_DCA'][$table];
 
-        if (isset($dca['fields'][$field]['eval']['flatpickr']) && \is_array($dca['fields'][$field]['eval']['flatpickr'])) {
+        if (
+            isset($dca['fields'][$field]['eval']['flatpickr'])
+            && \is_array($dca['fields'][$field]['eval']['flatpickr'])
+            && isset($dca['fields'][$field]['eval']['flatpickr']['options'])
+            && \is_array($dca['fields'][$field]['eval']['flatpickr']['options'])
+        ) {
```

Comment:
Redundant. You also can check nested array indices with isset, so

`isset($dca['fields'][$field]['eval']['flatpickr']['options']) && \is_array($dca['fields'][$field]['eval']['flatpickr']['options']`

is enough.